pubspec.yaml

```diff
@@ -61,7 +61,7 @@ dev_dependencies:
     sdk: flutter
   integration_test:
     sdk: flutter
-  msix: ^3.6.2
```

Comment:
```
Because msix >=3.0.0-beta.1 depends on cli_dialog ^0.5.0 which depends on dart_console ^1.0.0, msix >=3.0.0-beta.1 requires dart_console ^1.0.0.
And because dart_console >=1.0.0 depends on ffi ^1.0.0, msix >=3.0.0-beta.1 requires ffi ^1.0.0.
And because libedax4dart 5.3.1 depends on ffi ^2.0.1 and no versions of libedax4dart match >5.3.1 <6.0.0, msix >=3.0.0-beta.1 is incompatible with libedax4dart ^5.3.1.
So, because pedax depends on both libedax4dart ^5.3.1 and msix ^3.6.2, version solving failed.
pub get failed (1; So, because pedax depends on both libedax4dart ^5.3.1 and msix ^3.6.2, version solving failed.)
```